cyberark/conjur
# add page section anchors with anchor js

#### What does this pull request do?

Adds page section anchors for ease of navigation and reference to anchored sections.
While here, this PR also nudges the tooltip-text closer to highlight code block paste board icon.

See Screenshots below!

#### Where should the reviewer start?

Small diff made up of the following:

+ addition of `anchor.js` library file
+ link to `anchor.js` library file in `html` head template
+ custom script to initialise `anchor.js` instances

#### How should this be manually tested?

+ Build and serve the `conjur.org` site on your computer` as per https://github.com/cyberark/conjur/blob/master/docs/README.md
+ Visit the running application and confirm that when you hover on a section header and anchor appears to the right and when you click on it the url changes to reflect the anchor i.e. `http://localhost/path/to/current/page#section-identifier`. Below are some suggestions (assuming the site is running on `localhost:4000`). It's worth comparing `localhost:4000` against `conjur.org` to see the value in this change.
  - http://localhost:4000/tutorials/policy/applications.html
  - http://localhost:4000/get-started/install-conjur.html

#### Screenshots (if appropriate)

- page section anchors ![image](https://user-images.githubusercontent.com/8653164/31436757-caf21fb6-ae7a-11e7-8378-948db773b0e2.png)

- nudged tooltip-text ![image](https://user-images.githubusercontent.com/8653164/31437903-db8be4e8-ae7e-11e7-9658-d21b746bb5e0.png)

consult https://milanaryal.com/adding-hover-anchor-links-to-header-on-github-pages-using-jekyll/ for a short tutorial on anchor.js
